src/api.ts

```diff
@@ -240,31 +240,42 @@ export async function migrateList(
  * Check the filter condition and apply filter if required.
  *
  * @param {ConnectionReference[]} connections
- * @param {string} [filterConnectionId]
+ * @param {string} [filterConnectionIds]
```

Comment:
Let's rename this param to `connectionIds` so that it is more readable. The function name already says `filterConnectionsAsRequired`?

```suggestion
 * @param {string} [connectionIds]
```